lab_1_classify_by_unigrams/main.py

```diff
@@ -12,6 +12,16 @@ def tokenize(text: str) -> list[str] | None:
     :return: a list of lower-cased tokens without punctuation
     """
 
+    if type(text) == str:
```

Comment:
consider using `isinstance`

Reply:
another good practice would be to immediately escape the function when incorrect value is encountered, it makes the function easier to read